Week-2/Session-4/BirthdayMessageCreator.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Birthday Message Creator</title>
+    <style>
+        body {
+            position: relative;
+        }
+        /* Why do i have to define this ?
+        Without this css block my absolute positioning doesn't work */
```

Comment:
Absolute positioning always works based off the closest relative based parent. The why is this necessary is just how the CSS functions.

Reply:
Noted